src/canmatrix/cmjson.py

```diff
@@ -199,32 +200,42 @@ def load(f, **options):
                 newframe.extended = 0
 
             for signal in frame["signals"]:
-                if "is_big_endian" in signal and signal["is_big_endian"]:
+                if signal.get("is_big_endian", False):
                     is_little_endian = False
                 else:
                     is_little_endian = True
-                if "is_float" in signal and signal["is_float"]:
+
+                if signal.get("is_float", False):
                     is_float = True
                 else:
                     is_float = False
-                if "is_signed" in signal and signal["is_signed"]:
+
+                if signal.get("is_signed", False):
                     is_signed = True
                 else:
                     is_signed = False
-                newsignal = Signal(signal["name"],
+                newsignal = canmatrix.Signal(signal["name"],
                                    startBit=signal["start_bit"],
                                    size=signal["bit_length"],
                                    is_little_endian=is_little_endian,
                                    is_signed=is_signed,
+                                   is_float = is_float,
                                    factor=signal["factor"],
                                    offset=signal["offset"])
-                if "unit" in signal and signal["unit"]:
+
+                if signal.get("min", False):
```

Comment:
```suggestion
                if signal.get("min") is not None:
```

What about a `0` minimum?  It'll be falsey...